src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
The only place `force_hex` is used is here, where it also checks whether it is a 3-character currency code: https://github.com/ripple/ripple-lib/blob/develop/src/core/currency.js#L366

So I think it should be safe to make this change. Does that look right?